Do not build local copies of any of the dependencies on macOS.
(We use the MacPorts versions of them.)
--- lib/Global.cmake.orig	2017-06-05 11:30:44.000000000 -0500
+++ lib/Global.cmake	2018-03-15 11:05:11.000000000 -0500
@@ -9,12 +9,8 @@
 link_directories(${LocalPrefix}/lib)
 
 # Set flags indicating what we should build
-if (APPLE)
-  set(BUILD_JSONCPP true)
-  set(BUILD_XIPH true)
-  set(BUILD_PROTOBUF true)
-  set(BUILD_LIBPNG true)
-elseif (WINDOWS)
+if (WINDOWS)
+  set(BUILD_PHYSFS true)
   set(BUILD_JSONCPP true)
   set(BUILD_ZLIB true)
   set(BUILD_LIBPNG true)
